src/main/drivers/display_ug2864hsweg01.c

```diff
@@ -181,35 +183,61 @@ static bool i2c_OLED_send_cmd(busDevice_t *bus, uint8_t command)
     return i2cWrite(bus->i2c.device, bus->i2c.address, 0x80, command);
 }
 
+static bool i2c_OLED_send_cmdarray(busDevice_t *bus, const uint8_t *commands, size_t len)
+{
+    for (size_t i = 0 ; i < len ; i++) {
+        if (i2c_OLED_send_cmd(bus, *commands++)) {
```

Comment:
`commands[i]` is slightly better when used in `for` 
Or use `while(len--)` idiom